dspace-spring-rest/src/main/java/org/dspace/app/rest/model/MetadataValueRest.java

```diff
@@ -26,7 +25,7 @@
 
     int confidence;
 
-    @JsonProperty(access = Access.READ_ONLY)
+    @JsonIgnore
```

Comment:
I pause at this point in my review to wonder why this is suppressed from the JSON.  I see that there was some discussion on this in the Rest Contract.  Perhaps an additional comment would be useful here.